provisioning/miniprov/hare_mp/main.py

```diff
@@ -354,6 +361,56 @@ def start(args):
         start_hax_and_consul_without_systemd(url, utils)
 
 
+def start_mkfs(utils: Utils, hostname: str, hare_config_dir: str):
+    # TODO: path needs to be updated according to the new conf-store key
+    motr_config_dir = '/etc/motr'
+    sysconfig_dir = '/etc/sysconfig/'
+    os.makedirs(motr_config_dir, exist_ok=True)
+    shutil.copy(f'{hare_config_dir}/confd.xc', motr_config_dir)
+    src = f'{hare_config_dir}/sysconfig/motr/{hostname}'
+    for file in os.listdir(src):
+        shutil.copy(os.path.join(src, file), sysconfig_dir)
+    machine_id = utils.provider.get_machine_id()
+    MOTR_NODE_UUID = str(uuid.UUID(machine_id))
+    contents = f"MOTR_NODE_UUID='{MOTR_NODE_UUID}'"
+    save(f'{sysconfig_dir}/motr-kernel', contents)
+
+    generator = Generator(hostname, hare_config_dir,
+                          kv_file=f'{hare_config_dir}/consul-kv.json')
+    cmd = '/usr/libexec/cortx-motr/motr-mkfs'
+    # start mkfs for confd, ios services
+    for svc in ('confd', 'ios'):
+        svc_fids = generator.get_svc_fids(svc)
+        for fid in svc_fids:
+            if svc == 'ios':
+                execute([cmd, fid, '--conf', f'{motr_config_dir}/confd.xc'])
+            else:
+                execute([cmd, fid])
+
+
+def init(args):
+    try:
+        url = args.config[0]
+        utils = Utils(ConfStoreProvider(url))
+        stop_event = Event()
+        local_dir = '/var/hare'
```

Comment:
This path also needs to be fixed. Please sync up with @SwapnilGaonkar7 .

Reply:
changed as @SwapnilGaonkar7  suggested